test/hotspot/jtreg/compiler/ciReplay/CiReplayBase.java

```diff
@@ -159,7 +176,7 @@ public boolean generateReplay(boolean needCoreDump, String... vmopts) {
             throw new Error("Can't create replay: " + t, t);
         }
         if (needCoreDump) {
-            String coreFileLocation = getCoreFileLocation(crashOutputString);
+            String coreFileLocation = getCoreFileLocation(crashOutputString, pid);
```

Comment:
Please see the comments I just added to JDK-8259501. This test should be using the new core file support in CoreUtils.java. I think this means you don't need any of the ProcessTools.java changes.

Reply:
Thanks for the pointer. I pushed a new change that refactors the test so it takes advantage of CoreUtils.